sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/caches/RxPartitionKeyRangeCache.java

```diff
@@ -247,5 +246,20 @@ public RxPartitionKeyRangeCache(RxDocumentClientImpl client, RxCollectionCache c
                     }, 1).collectList();
         });
     }
-}
 
+    /**
+     * Force refreshes the partition range cache
+     * @param metadataDiagnosticsContext the metadata diagnostics context
+     * @param collectionRId the collectionRid
+     * @return a void Mono
+     */
+    public Mono<Void> forceRefresh(MetadataDiagnosticsContext metadataDiagnosticsContext, String collectionRId) {
```

Comment:
Is this needed or can't the policy invoke `tryGetOverlappingRangesAsync` with forceRefresh and the full range?

Reply:
Actually that works. Moved to tryGetOverlappingRangesAsync